Sources/SnowManLogic/ViewLogic/Receivable.swift

```diff
@@ -0,0 +1,4 @@
+public protocol Receivable {
```

Comment:
I think there could be a more descriptive name than Receivable, based on what the protocol actually says it will do.